### ISSUE TITLE
Critical Security Issue

### ISSUE THREAD

**Issue by shaneonabike** [pressflow/7#74]
I noticed that Drupal 7/8 Core is release a security patch that is requiring everyone upgrade immediately once the patch is released. I'm wondering if we couldn't integrate that and release a new release for Pressflow quickly also?

https://www.drupal.org/psa-2018-001

**Comment by mparker17** [comment]
I have just checked my local environments to ensure that they will be ready to create a merge request when the time comes.

For reference, and/or if you'd like to try this on your own, here's what I do.

-----

To prepare: 

1. Fork pressflow/7 to my Github account (my fork is https://github.com/mparker17/7 )
2. Set up my local environment as follows: ```
git clone --recursive git@github.com:mparker17/7.git
git remote add upstream https://github.com/pressflow/7.git
git remote add drupal https://git.drupal.org/project/drupal.git
git fetch --all -t -p
git merge upstream/master
git push origin master
```

------

Once release day comes along, I generate a release as follows: 

1. Run ```
git fetch --all -t -p
git checkout -b pressflow-7.58 # or whatever the release version number happens to be
git merge 7.58 # or whatever the tag happens to be named in the drupal repo
# resolve any merge conflicts if there are any
git push --set-upstream origin pressflow-7.58 # or whatever the release version number happens to be
```
2. Create a pull request in the main pressflow repository by going to: https://github.com/pressflow/7/compare/master...mparker17:pressflow-7.58?expand=1 — in the pull request I tag the other pressflow maintainers for their approval.

**Comment by mparker17** [comment]
Fix pushed to #75

**Comment by generalredneck** [comment]
I totally missed pushing the tag... I'm sorry bout that... the tag is now released too for those using tags to pull in updates.
